pkg/logentry/stages/multiline.go

```diff
@@ -203,11 +203,15 @@ func (m *multilineStage) flush(out chan Entry, s *multilineState) {
 		}
 		return
 	}
-
+	// copy extracted data.
+	extracted := make(map[string]interface{}, len(s.startLineEntry.Extracted))
+	for k, v := range s.startLineEntry.Extracted {
+		extracted[k] = v
+	}
 	collapsed := Entry{
-		Extracted: s.startLineEntry.Extracted,
+		Extracted: extracted,
```

Comment:
That's interesting. I thought `Extracted` would be a value not a reference. *few minutes googling later*

Ok, maps are references, see https://blog.golang.org/maps:

> Map types are reference types, like pointers or slices, and so the value of m above is nil; it doesn't point to an initialized map.

:facepalm: 

So either we send a copy across the channel. Or we reset the entry but that would be a little bit more complicated.